deploy/006-deploy-mock-pools.ts

```diff
@@ -9,43 +9,13 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { deployments, getNamedAccounts }: any = hre;
   const { deploy } = deployments;
   const { deployer, proxyAdmin } = await getNamedAccounts();
-  //=======================
-  // DEPLOY MOCK TOKENS
-  //========================
-  await deploy("MockBTC", {
-    from: deployer,
-    contract: "MockToken",
-    args: ["Bitcoin", "BTC", 8],
-    log: true,
-    autoMine: true, // speed up deployment on local network (ganache, hardhat), no effect on live networks
-  });
-
-  const wBTC = await ethers.getContract("MockBTC");
-
-  await deploy("MockDAI", {
-    from: deployer,
-    contract: "MockToken",
-    args: ["MakerDAO", "DAI", 18],
-    log: true,
-    autoMine: true,
-  });
 
-  const DAI = await ethers.getContract("MockDAI");
+  const BNX = await ethers.getContract("MockBNX");
```

Comment:
We should refactor this now that Pyth pusher is working. When we are refactoring it we should also create a hardhat task to add pools and markets rather than hardcode them in this script?

Reply:
how is that connected to pyth pusher?

Reply:
we will refactor this task to be universal and add pools per configuration json. In this json we will include all the metadata around pools (markets tokens and etc)

Reply:
The pyth pusher prevented us from deploying the using real testnet tokens was my understanding